src/coreclr/jit/importercalls.cpp

```diff
@@ -10962,6 +11018,19 @@ NamedIntrinsic Compiler::lookupNamedIntrinsic(CORINFO_METHOD_HANDLE method)
                             {
                                 result = NI_System_Runtime_CompilerServices_RuntimeHelpers_GetMethodTable;
                             }
+                            else if (strcmp(methodName, "Await") == 0)
+                            {
+                                result = NI_System_Runtime_CompilerServices_RuntimeHelpers_Await;
+                            }
+                            else if (strcmp(methodName, "AsyncSuspend") == 0)
+                            {
+                                result = NI_System_Runtime_CompilerServices_RuntimeHelpers_AsyncSuspend;
+                            }
+                            else if (strcmp(methodName, "get_RuntimeAsyncViaJitGeneratedStateMachines") == 0)
```

Comment:
Does the get_RuntimeAsyncViaJitGeneratedStateMachines still exist?

Reply:
No, let me delete this downstream too... One of the unported changes in #113976.